### DISCUSSION TITLE
手机端应用搜索功能无法使用

### DISCUSSION THREAD

**Discussion by LSQ-Code** [community/community#131280] [Other Feature Feedback, Questions, & Ideas]
是本来就没有做这个功能，还是说只是我这里有bug，点击右上角搜索图标没有任何事情发生

**Comment by gaomingzhao666** [comment]
**你好老哥。**

**我没太看懂你的问题描述，如果这是你做的app那么你可以点击搜索按钮后查看浏览器的console版块看是否有报错，并将报错信息谷歌查询一下解决办法。**

**如果你点击了按钮后console未出现任何信息，那么可能是以下原因：**
- 搜索功能并没有在代码层面实现
- 你的项目未过编译
- 你的项目没有安装好项目所需的依赖
- 你的项目还没有启动就直接点开了localhost地址
- 项目在前端并未给此按钮添加任何事件或者按钮使用的http接口请求有问题

**如果你点击了按钮后console有错误信息，那么可能有以下错误信息：**
- CORS 跨域报错，如果你的浏览器或者项目没有设置好跨域请求，在点击按钮后发送http请求就会产生跨域，此时自然无法使用功能
- 你输入的关键字不合规定，如输入了空白字符串等

**若检测到错误信息你可以去谷歌查询后根据相关解决办法来解决。
如果可以的话，你可以提供一个精简的 [Reproduction](https://en.wikipedia.org/wiki/Reproduction) 不然我很难帮助你，如果可以的话你可以使用英语描述问题，这样回复你的人也会更多一些。**

**如果我的回答解决了你的问题请标注我的回复为answer，谢谢老哥！**

**Comment by ossanna16** [comment]
Hi @LSQ-Code , thanks for participating!

👋 and thank you for being a part of our community. To ensure a safe and supportive environment, we look to our[Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) which states all commununcations here on the Community needs to be in English. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
